src/applications/edu-benefits/1995/helpers.jsx

```diff
@@ -1,3 +1,29 @@
 // 1995-STEM related
-export const displayActiveDutyStem = form =>
+import environment from 'platform/utilities/environment';
+
+const isEdithNourseRogersScholarship = form =>
   form.isEdithNourseRogersScholarship;
+
+export const isChapter33 = form =>
+  form.benefit === 'chapter33' || form.benefit === 'fryScholarship';
+
+export const isEligibleForEdithNourseRogersScholarship = form =>
```

Comment:
Unit tests for all of these exported functions would be nice to have.

Reply:
Added unit tests